vertx-web/src/main/java/io/vertx/ext/web/handler/sse/SSEConnection.java

```diff
@@ -0,0 +1,68 @@
+package io.vertx.ext.web.handler.sse;
+
+import io.vertx.codegen.annotations.Fluent;
+import io.vertx.codegen.annotations.GenIgnore;
+import io.vertx.codegen.annotations.VertxGen;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.ext.web.handler.sse.impl.SSEConnectionImpl;
+
+import java.util.List;
+
+@VertxGen
+public interface SSEConnection {
+
+	static SSEConnection create(RoutingContext context) {
+		return new SSEConnectionImpl(context);
+	}
+
+	@Fluent
+  SSEConnection forward(String address);
+
+	@Fluent
+  SSEConnection forward(List<String> addresses);
+
+	@Fluent
+  SSEConnection reject(int code);
+
+	@Fluent
+  SSEConnection reject(int code, String reason);
+
+	@Fluent
+  SSEConnection comment(String comment);
+
+	@Fluent
+  SSEConnection retry(Long delay, List<String> data);
+
+	@Fluent
+  SSEConnection retry(Long delay, String data);
+
+	@Fluent
+  SSEConnection data(List<String> data);
```

Comment:
What's the use case for this?

Reply:
Sending multiple lines at once?

Reply:
I was wondering why, as a user, would you want to split content into multiple `data` events?

Reply:
🤷🏻‍♂️
In the docs [here](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events) there's an example, but not really obvious:

```
data: Here's a system message of some kind that will get used
data: to accomplish some task.
```

We can remove it if you think it's not accurate. It can easily be achieved by invoking `data()` multiple times. No worries.

Reply:
I understood that you can send multiple _data_ messages but I was wondering if there are cases when you need to split content.

Reply:
Actually, invoking `data(...)`  multiple times doesn't really achieve the same thing.
Since invoking ` data(...)`  finishes the message with `\n\n`.
But in the example from the docs both ` data:` are separated with just one carriage return, which means on the client side they'll be merged into one single String.

So this point is NOT addressed yet. We need to figure out how to mimic this example on the server, and fix the EventSource client so that it merges multiple ` data:` lines into one single String.

Reply:
There are different ways we can address this. One could be, in `SSEConnection`  to have:
* a generic: `SSEConnection data(String msg, boolean endPacket)`  method
* rewrite `SSEConnection data(String msg)`  method to be ` default SSEConnection data(String msg) { return data(msg, true); }` 
* And then let the user invoke ` data("...", false);data("......", false);data("last message");` 

What do you think of this @tsegismont if you agree, it'll allow me to test the ` EventSource`  counterpart (merging multi-line messages into a single String) as mentioned in the specs very easily, since the multi-line functionnality would be supported on the server.

Reply:
I implemented this, which allowed me to easily write a unit test for the EventSource client ability to merge multiline data into a Single String. Let me know what you think about it.